docs/tutorials/05_torch_connector.ipynb

```diff
@@ -262,120 +329,142 @@
   },
   {
    "cell_type": "markdown",
-   "id": "moral-berry",
+   "id": "fb029de7",
    "metadata": {},
    "source": [
-    "The red circles indicate wrongly classified data points."
+    "We can see that the model can generally achieve an accuracy over 75% (depending on the dataset). The red circles indicate wrongly classified data points."
```

Comment:
Can we be sure we can achieve 75%?

Reply:
Actually, I did multiple runs this morning and no, we cannot be sure. I removed this line on the new commit.